pkg/tracegen/templated.go

```diff
@@ -50,6 +50,10 @@ type SpanDefaults struct {
 	Attributes map[string]interface{} `js:"attributes"`
 	// RandomAttributes random attributes generated for each span.
 	RandomAttributes *AttributeParams `js:"randomAttributes"`
+	// Random events generated for each span
+	EventDefaults EventDefaults `js:"eventDefaults"`
+	// Random links generated for each span
+	LinkDefaults LinkDefaults `js:"linkDefaults"`
```

Comment:
Rename to `RandomEvents` and `RandomLinks` in order to be more consistent with the existing `RandomAttributes`